examples/python/ToF/tof_raw_rvc4.py

```
@@ -0,0 +1,244 @@
+    raw_in_q = tof_base.rawInput.createInputQueue()
+    depth_q  = tof_base.depth.createOutputQueue()
+
+    return pipeline, raw_in_q, depth_q
+
+
+def verify_saved_files(out_dir: str, expected_count: int) -> bool:
+    depth_files = sorted(f for f in os.listdir(out_dir) if f.startswith("depth_") and f.endswith(".npy"))
+
+    print(f"\n[Verify] Expected {expected_count} file(s), found {len(depth_files)}")
+    if len(depth_files) < expected_count:
+        print(f"[FAIL] Not enough depth files saved.")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)